### PR TITLE
Added new translatable keys for mod description in Mod Menu

### PR DESCRIPTION
**Pull request by SyberiaKey**
While looking at localization files for other mods, I found out that Mod Menu has its own translatable keys for mods descriptions:
- "_modmenu.summaryTranslation.dynamiccrosshair_" - short description
- "_modmenu.descriptionTranslation.dynamiccrosshair_" - detailed description

I added these keys to all localization files so we can use it :)

### CODE REVIEW

**Comment by Crendgrim** [comment]
Interesting!

I would change the English summary string to:
`  "modmenu.summaryTranslation.dynamiccrosshair": "Contextual crosshair style",`

so that it's short and succinct, and doesn't get eclipsed in its box. Looks a lot neater. The Russian translation would need to be updated accordingly.

**Comment by SyberiaKey** [comment]
> Interesting!
> 
> I would change the English summary string to: ` "modmenu.summaryTranslation.dynamiccrosshair": "Contextual crosshair style",`
> 
> so that it's short and succinct, and doesn't get eclipsed in its box. Looks a lot neater. The Russian translation would need to be updated accordingly.

Done.

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "${version}",
 
   "name": "Dynamic Crosshair",
-  "description": "Dynamic Crosshair that changes shape based on what is targeted",
+  "description": "Contextual crosshair style",
```
Comment (Crendgrim) [inline src/main/resources/fabric.mod.json:7]: Do you know where this string is used? Is it worth to shorten it here?

Comment (SyberiaKey) [reply src/main/resources/fabric.mod.json:7]: Not quite sure, maybe it's sort of system mod description

Comment (SyberiaKey) [reply src/main/resources/fabric.mod.json:7]: Maybe that string uses if "modmenu.summaryTranslation.dynamiccrosshair" key not found in localization file (or there's no localization file for this language)

Comment (Crendgrim) [reply src/main/resources/fabric.mod.json:7]: From my quick testing, that string is indeed used if there is no `summaryTranslation` string. However, once that `summaryTranslation` string exists in `en_us.json`, other languages appear to automatically pick that one up.

So I believe we can keep this more verbose (in its old form).

Comment (SyberiaKey) [reply src/main/resources/fabric.mod.json:7]: Ok, one sec

**Comment by Crendgrim** [comment]
@IceeAlin Hi, just a ping in case you'd like to translate these new keys to Chinese as well :)

**Pull request merged**